src/Diff/Internal/AliasGroupListPatcher.php

```diff
@@ -37,7 +36,7 @@ public function patchAliasGroupList( AliasGroupList $groups, Diff $patch ) {
 	}
 
 	/**
-	 * @see MapPatcher
+	 * @see Diff\Patcher\MapPatcher
```

Comment:
`@return`, `@param` and `@var` describe actual behaviour of the code, while `@see` is more like a `<a href>` to guide the reader. Considering this the difference makes sense to me.